primitives/src/runtime_api.rs

```diff
@@ -220,6 +224,10 @@ sp_api::decl_runtime_apis! {
 
 		/// Returns all onchain disputes.
 		#[api_version(3)]
-		fn disputes() -> Vec<(v2::SessionIndex, v2::CandidateHash, v2::DisputeState<v2::BlockNumber>)>;
+		fn disputes() -> Vec<(SessionIndex, CandidateHash, DisputeState<BlockNumber>)>;
+
+		/// Returns execution parameters for the session.
+		#[api_version(3)]
+		fn session_executor_params(session_index: sp_staking::SessionIndex) -> Option<vstaging::ExecutorParams>;
```

Comment:
sorry I missed this, but why is this still in `vstaging` if it's stable since api version 3?

Reply:
```suggestion
		fn session_executor_params(session_index: SessionIndex) -> Option<vstaging::ExecutorParams>;
```

Reply:
Well, I cannot say I have a clear understanding of how versioning goes, but [here](https://github.com/paritytech/polkadot/pull/6161#discussion_r1087986779) @eskimor says it's okay so I just assume it's okay :)

Iiuc, we needed to push it to v3 because we needed to push primitives to v3, because we were changing `SessionInfo`, which is v2, and there was no way to preserve v2 together with v3. Now, as we've given up the idea to integrate it into `SessionInfo`, we're no more bound by those restrictions, and we can first release it on Westend and Rococo only, so the base API version remains v2, mainnets remain on v2, and only v3 nets get the upgrade, making it staging.

Reply:
gotcha, that makes sense!